feat(crowdfundings): Migrate memberships, add column gracePeriodInterval

patrickvenetz opened the pull request:
Replaces `UNCANCELLED_GRACE_PERIOD_DAYS` with `gracePeriodInterval` attribute on `memberships` table. Defaults to 14 days.

- [ ] Extend grace interval for recipients of a payment slip

patte commented on servers/republik/modules/crowdfundings/graphql/resolvers/Membership.js:
```diff
@@ -91,8 +90,12 @@ module.exports = {
-        return moment(getLastEndDate(periods))
-          .add(UNCANCELLED_GRACE_PERIOD_DAYS, 'days')
+        const graceEndDate = moment(getLastEndDate(periods))
+        Object.keys(membership.gracePeriodInterval).forEach(key => {
```
Thinking about it now that I read all the code, I think a small lib to add/subtract pg-intervals to moments would be handy.

patrickvenetz replied:
Done so, see https://github.com/orbiting/backends/pull/201

patrickvenetz commented on servers/republik/migrations/sqls/20190125150100-memberships-grace-period-interval-up.sql:
```diff
@@ -0,0 +1,3 @@
+ALTER TABLE "memberships"
+  ADD COLUMN "gracePeriodInterval" interval NOT NULL DEFAULT '14 days'::interval
```
Renamed.

patrickvenetz commented:
@patte Please revisit.

- Renamed `gracePeriodInterval` to `graceInterval`
- Replaced whitch-crafty Postgres interval add with `applyPgInterval` util
